### DISCUSSION TITLE
where to find kubernetes compatibility?

### DISCUSSION THREAD

**Discussion by smyja** [fluxcd/flux2#4697] [Q&A] [answered]
I am looking for a compatibility matrix for flux showing the various versions and its supporting kubernetes versions for the releases.

**Comment by souleb** [comment]
See https://fluxcd.io/flux/releases/#kubernetes-supported-versions.

Also from a given cli version `flux check --pre` will tell you if your cluster is on a supported version.

**Comment by stefanprodan** [comment, marked as answer]
For the latest Flux version the Kubernetes supported versions are listed here: https://fluxcd.io/flux/installation/#prerequisites

Also each minor release of Flux contains the Kubernetes supported versions in the changelog https://github.com/fluxcd/flux2/releases

**Reply by smyja** [reply]
Thanks, that's exactly what i wanted.
